Lib/test/test_call.py

```diff
@@ -194,6 +195,27 @@ def test_varargs13_kw(self):
         msg = r"^classmethod\(\) takes no keyword arguments$"
         self.assertRaisesRegex(TypeError, msg, classmethod, func=id)
 
+    def test_varargs14_kw(self):
+        msg = r"^product\(\) takes at most 1 keyword argument \(2 given\)$"
+        self.assertRaisesRegex(TypeError, msg, itertools.product, 0, a=1, b=2)
```

Comment:
I think it would be better to use names of supported arguments (and one unsupported).